chef/cookbooks/monasca/attributes/default.rb

```diff
@@ -41,3 +41,8 @@
 default[:monasca][:log_agent][:user] = "monasca-log-agent"
 default[:monasca][:log_agent][:group] = "logstash"
 default[:monasca][:log_agent][:debug] = monasca_debug
+
+default[:monasca][:ha][:enabled] = false
+# Ports to bind to when haproxy is used for the real ports
+default[:monasca][:ha][:ports][:plain] = 5580
```

Comment:
Yes, but please find a port that doesn't clash with anything else. As far as I know we do not have a fixed convention for that...